CSS/explore.css

```diff
@@ -285,6 +285,9 @@ span.labelOncheck {
 .mobileSec {
   display: none;
 }
+.colorbtnClick {
```

Comment:
Use proper naming conventions
**.colorbtnClick** this is not meaningful name and it can be specific for single button, can not be re-usable